### PR TITLE
Adds the ability to specify the order statuses for the navigation count

### PR DESCRIPTION
**Pull request by glennjacobs**
You can now add the statuses the admin panel should use for the order count.

### CODE REVIEW

File: packages/admin/config/panel.php [hunk 1]
```diff
@@ -35,4 +35,17 @@
...
+    |--------------------------------------------------------------------------
+    | Navigation counts
+    |--------------------------------------------------------------------------
+    |
+    | The admin panel will show a count of orders in the left navigation.
+    | This is based upon specific order statuses. You can define the statuses
+    | to include in the count below.
+    |
+    */
+    'order_count_statuses' => ['in-progress'],
```
Comment (alecritson) [inline packages/admin/config/panel.php]: @glennjacobs Lunar doesn't ship with an `in-progress` status

Comment (glennjacobs) [reply packages/admin/config/panel.php]: Good point haha!

**Comment by alecritson** [comment]
Should we format this so it looks a bit nicer?
image

**Comment by glennjacobs** [comment]
> Should we format this so it looks a bit nicer? image

Do you mean the number format? So it's `558,265`?

**Comment by alecritson** [comment]
> > Should we format this so it looks a bit nicer? image
> 
> Do you mean the number format? So it's `558,265`?

Yeah, I guess it would need to take into account Locale as not everyone format numbers like that.

**Comment by glennjacobs** [comment]
> > > Should we format this so it looks a bit nicer? image
> > 
> > 
> > Do you mean the number format? So it's `558,265`?
> 
> Yeah, I guess it would need to take into account Locale as not everyone format numbers like that.

Exactly, this could turn a simple PR into a much bigger issue, as then we'd have to check every count format in the admin panel.

**Review by alecritson — APPROVED** [review]

**Pull request merged**